README.md

```diff
@@ -89,4 +89,8 @@ to build in prod use command
 
  
 Website address https://kp01012022.azurewebsites.net/
```

Comment:
The website is currently not working. You will need to fix this before we can check the functionality is working as expected
![image](https://user-images.githubusercontent.com/92427951/207307940-491419f3-f939-4b99-aa1a-80b570da6830.png)

Reply:
There is more info to confirm that the site is not working currently 
https://kp01012022.scm.azurewebsites.net/

![image](https://user-images.githubusercontent.com/92427951/207308765-4d2579df-2fc9-41b4-af0d-7d3748ef9760.png)